guides-markdown/wallets.md

```diff
@@ -0,0 +1,357 @@
+
+#### Keys to the Game
+
+If you want to transact with bitcoin, you'll need keys. Each bitcoin address is associated with a particular key, and wallets are made up of many different keys. Keys consist of both a private key and a public key. The private key is required for spending and is extremely sensitive information, while a public key can be used to receive bitcoins and monitor a particular address. If you want to learn more about how this works, read up on [Public-Key Cryptography](https://en.wikipedia.org/wiki/Public-key_cryptography).
+
+#### HD vs Non-HD, you mean like TVs?
+
+You may have seen references to "HD" wallets and wondered what that means. HD in this context does not mean "high definition," as I assumed it did at first, but rather "heirarchal deterministic." An HD wallet takes a *heirarchy* of keys in order and makes it so any key in that sequence can be *determined* by the one before it. This means that if you can produce the first key in the heirarchy, you can then generate a practically unlimited number of subsequent keys. The specification for HD wallets as implemented in bcoin is defined by [Bitcoin Improvement Proposal (BIP) 32](https://github.com/bitcoin/bips/blob/master/bip-0032.mediawiki).
+
+Non-HD wallets, on the other hand, contain keys that are unrelated to one another. Backing up such a wallet means each key must be preserved individually. Not only is this more cumbersome, but it means that backups can quickly become out of date as new keys are added to the wallet. With an HD wallet, as long as you hold on to the seed - the data needed to recreate the first key - you will be able to recover every other key.
+
+While bcoin uses HD wallets, it does allow you to import individual keys into a wallet. This can be a handy feature in certain cases, but it means you'll need to backup any imported keys separately as they will not be recoverable simply by using your seed.
+
+But what exactly is the seed for an HD wallet? It can come in several forms, but bcoin implements [BIP39](https://github.com/bitcoin/bips/blob/master/bip-0039.mediawiki) which enables seeds to be represented by a mnemonic of twelve common words. This means your seed can be easily spoken, written down, or perhaps even memorized. But be careful! Your seed can be used to recover and spend everything in your HD wallet (except for the aforementioned imported keys), so treat it like you would an actual wallet with cash in it.
```

Comment:
You can actually do more than 12 words. when creating a mnemonic seed, the length of the list is determined by the bits of entropy (http://bcoin.io/docs/module-hd.Mnemonic.html). This defaults to 128 bits which generates 12 words, but if you pass in `bits` option of `256` it will do a list of 24.

Reply:
Good point, I was aware of this actually but am so used to 12 words as the standard. I'll mention the possibility for increased entropy.